tests/docker-compose.yml

```diff
@@ -8,6 +8,10 @@ services:
       - TZ=Europe/Moscow
     ports:
       - "127.0.0.1:9000:9000"
+      - "127.0.0.1:9440:9440"
```

Comment:
We should add port 9440 in development docs: docs/development.rst:48

Reply:
Fixed